source/learn/intrinsics/ARRAY_index.md

```diff
@@ -0,0 +1,70 @@
+# Properties and attributes of arrays
```

Comment:
Can we move this page to `source/learn/intrinsics/array/index.md` or `source/learn/intrinsics/array.md`?